### PR TITLE
PAINTROID-154 Add "Auto" to crop icon

### PR DESCRIPTION
**Pull request by FromG**
added "Auto" to crop icon.

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by ThorstenBandel — APPROVED** [review]
Look good to me

File: Paintroid/src/main/res/values/string.xml [hunk 1]
```diff
@@ -114,6 +114,7 @@
     <string name="transform_tool_auto_crop_text">crop/enlarge</string>
     <string name="transform_width_text">Width</string>
     <string name="transform_height_text">Height</string>
+    <string name="transform_auto_crop_text" translatable="false">Auto</string>
```
Comment (wslany) [inline Paintroid/src/main/res/values/string.xml:117]:
```suggestion
    <string name="transform_auto_crop_text">Auto</string>
```

This should also be translatable please.

**Review by wslany — CHANGES_REQUESTED** [review]
Thanks!

Please change the size and color of the "Auto" text below the crop symbol to those of the "Width" in the same line. The symbol shall also have the same color as the text (same grey). 
image

**Review by StanglTobias — COMMENTED** [review]
@wslany i would suggest to keep the Auto text and icon in black and rather change the texts of width and height to black because otherwise it looks like it would be disabled to me. Or we could move it to the other transform options

**Review by StanglTobias — APPROVED** [review]

**Comment by wslany** [comment]
> @wslany i would suggest to keep the Auto text and icon in black and rather change the texts of width and height to black because otherwise it looks like it would be disabled to me. Or we could move it to the other transform options

@StanglTobias you are right, and I agree. I checked the other tools, where such texts are usually in the light blue of the other settings texts and fields, and other elements are either also in the light blue or in black. See, e.g., 
image

(Actually, I think the circle and square at the bottom in the brush tool settings in the screenshot above should also be in the light blue color --- @ThorstenBandel what do you think? --- but that should be handled in a different ticket, and if so, changed in the other ticket for all relevant places in the tool settings).

In summary, @FromG please change the texts in the transform tool and the "Auto" button to the light blue color. The only element that should remain in grey is the line below the number. Sorry for this, but I think Tobias is right, and let's change it now while we still know where this can be changed.

File: Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml [hunk 2]
```diff
@@ -84,6 +89,7 @@
                 android:layout_height="wrap_content"
                 android:labelFor="@+id/pocketpaint_transform_height_value"
                 android:text="@string/transform_height_text"
+                android:textColor="#33B5E5"
```
Comment (StanglTobias) [inline Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml:92]: please use `android:textColor="?attr/colorAccent"`

File: Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml [hunk 3]
```diff
@@ -103,8 +109,9 @@
             <TextView
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:textSize="14sp"
-                android:text="@string/pixel" />
+                android:text="@string/pixel"
+                android:textColor="#33B5E5"
```
Comment (StanglTobias) [inline Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml:113]: please use android:textColor="?attr/colorAccent"

File: Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml [hunk 4]
```diff
@@ -72,8 +76,9 @@
...
```
Comment (StanglTobias) [inline Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml:80]: please use android:textColor="?attr/colorAccent"

File: Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml [hunk 5]
```diff
@@ -53,6 +56,7 @@
                 android:layout_height="wrap_content"
                 android:labelFor="@+id/pocketpaint_transform_width_value"
                 android:text="@string/transform_width_text"
+                android:textColor="#33B5E5"
```
Comment (StanglTobias) [inline Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml:59]: please use android:textColor="?attr/colorAccent"

File: Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml [hunk 6]
```diff
@@ -36,14 +36,17 @@
...
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:src="@drawable/ic_pocketpaint_tool_resize_adjust"
-                android:theme="@style/PocketPaintHighlightColorTheme"
-                android:contentDescription="@string/transform_tool_auto_crop_text" />
+                android:contentDescription="@string/transform_tool_auto_crop_text"
+                android:drawableTop="@drawable/ic_pocketpaint_tool_resize_adjust"
+                android:text="@string/transform_auto_crop_text"
+                android:textAppearance="@style/TextAppearance.AppCompat.Small"
+                android:textColor="#33B5E5"
```
Comment (StanglTobias) [inline Paintroid/src/main/res/layout/dialog_pocketpaint_transform_tool.xml:48]: please use android:textColor="?attr/colorAccent"

**Review by StanglTobias — CHANGES_REQUESTED** [review]

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!!
